base/utils/R/read.output.R

```diff
@@ -80,6 +80,32 @@ read.output <- function(runid, outdir, start.year = NA, end.year = NA, variables
       for (v in variables) {
         if (v %in% c(names(nc$var), names(nc$dim))) {
           newresult <- ncdf4::ncvar_get(nc, v)
+          # begin per-pft read
+          if("pft" %in% sapply(nc$var[[v]]$dim, `[[`, "name")){
+            # means there are PFT specific outputs we want
+            # parse pft names and match the requested
+            pft.string <- ncdf4::ncatt_get(nc, "PFT")
```

Comment:
When referencing pft/PFT, seem to be switching case between this line and the `if` right above. Is there a reason we're not consistent?

Reply:
Ok, that should be documented inline since I just demonstrated that someone else going to read/modify that code will miss that subtlety